assets/templates/sharing_discovery.html

```diff
@@ -0,0 +1,58 @@
+        </symbol>
+      </defs>
+    </svg>
+    <main role="application">
+      <section class="popup">
+        <div>
+          <div class="container">
+            <form method="POST" action="/sharings/discovery" class="login auth">
+              <input type="hidden" name="sharing_id" value="{{.SharingID}}" />
+              <input type="hidden" name="recipient_id" value="{{.RecipientID}}" />
+              <header>
+                <figure>
+                  <div class="svg-wrapper">
+                    <svg>
+                      <use xlink:href="#cozy-icon" />
+                    </svg>
+                  </div>
+                </figure>
+              <h1>{{t "Hello recipient email" .RecipientEmail}}</h1>
+              </header>
+              <div role="region">
+                  <input id="redirect" type="hidden" name="redirect" value="{{.Redirect}}" />
+                  <p class="help">{{t "Discovery URL help" .PublicName}}</p>
+                  <p class="line">
+                    <label for="URL"> {{t "Discovery URL field"}}</label>
+                    <input id="url" name="url" placeholder="{{t "Discovery URL placeholder"}}" type="text" autofocus="true" />
+                  </p>
+              </div>
+              <footer>
+                <div class="controls">
+                  <button type="submit">{{t "Discovery Submit"}}</button>
+                </div>
+              </footer>
+            </form>
+          </div>
+        </div>
+    </section>
+      </main>
+    </div>
+    <script src="/assets/scripts/async-submit.js"></script>
```

Comment:
If it is the case, you need to answer JSON to that endpoint. See https://github.com/cozy/cozy-stack/blob/master/web/auth/auth.go#L127-L159